### DISCUSSION TITLE
Recovering a deleted account

### DISCUSSION THREAD

**Discussion by dunhamsdecker** [mastodon/mastodon#20261] [Q&A] [answered]
Is there a way to recover a deleted account?  If so, what do I need to do to start the recovery process?

**Comment by MrGeneration** [comment, marked as answer]
I'm sure you can't. When deleting your own account in an instance the following warning is displayed:

![image](https://user-images.githubusercontent.com/6549061/201130384-f4bbbe39-654b-41aa-b888-41fb64cddaa8.png)


I double tabbed with a test account, even admins can't bring it back.
There might be manual database fiddling options to make Mastodon forget that the username is taken, but that might bring more issues than benefits.

**Reply by docum3nt** [reply]
But how can a suspended account be resinstated — there is no way to contact the admins once suspension has taken place. Is there some secret email address that can be used to apply for reinstatement?

**Reply by MrGeneration** [reply]
As I said: Reactivation and restoration of these accounts is not possible.

Most of the Mastodon instsances have an administrative contact on their about page.

**Reply by docum3nt** [reply]
Thanks very much.

**Reply by NickyBarness** [reply]
@MrGeneration 
Can you look at my question thx!

**Reply by docum3nt** [reply]
Unfortunately mastodon.ie/@mastodonie (admins) won't show me the page because it says my account is currently disabled. It just spins the spinner and won't finish loading, which means the contact address is shown redacted. The about page says "The requested page could not be rendered."
